### PR TITLE
R4R (I think): Create Working Group Page

### PR DESCRIPTION
**Pull request by Michelle-L**
Rough draft for new working group page

### CODE REVIEW

File: IBC_WORKING_GROUP.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# IBC Working Group
+
+Join and contribute to the IBC working group 
+
+The evolution of IBC requires cross-industry collaboration to support developing standards that are secure, open, and deployable. The IBC working group is a growing team of contributors supporting the standardization of blockchain development from protocol level to applications. 
+
+Join the IBC [Working Group](https://zc.vg/sf/giAEr) email updates
+
+Join the IBC Telegram channel
```
Comment (cwgoes) [inline IBC_WORKING_GROUP.md]: I am not a huge fan of using Telegram to coordinate or discuss, it is too ephemeral.

I would prefer a Discourse server, I think forums are much more conducive to long-form, considered discussion.

Is this something that would be possible? I would be quite willing to help set it up, administer/moderate, etc.

Perhaps a dedicated subsection of [forum.cosmos.network](https://forum.cosmos.network) could also be used, although I think there are advantages in branding it separately.

File: IBC_WORKING_GROUP.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+The founding IBC team
```
Comment (cwgoes) [inline IBC_WORKING_GROUP.md]: If she is interested, I would invite @ancazamfir to join, her spec reviews have been really helpful.

Comment (ancazamfir) [reply IBC_WORKING_GROUP.md]: Yes I would love to if everyone is ok with it.

**Comment by Michelle-L** [comment]
Here is what I think is a good example of what I am trying to recreate: https://github.com/ethereum/pm

File: IBC_WORKING_GROUP.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+Purpose: The evolution of IBC requires cross-industry collaboration to support developing standards that are secure, open, and deployable. The IBC working group is a growing team of contributors supporting the standardization of blockchain development from protocol level to applications. Join the IBC [Working Group](https://zc.vg/sf/giAEr) list for email and meeting updates.
+
+The IBC working group is technical in nature as the goal is to propose, implement and design templates for cross-chain interoperability.  
+
+Upcoming IBC working group calls (Dial in details are provided in the issue for the upcoming meeting)
+
+- November 14th, 2019 17:00 UTC 
+- November 28th, 2019 17:00 UTC
+- December 12th, 2019 17: 00 UTC
```
Comment (cwgoes) [inline IBC_WORKING_GROUP.md]:
```suggestion
- December 12th, 2019 17:00 UTC 
```

File: IBC_WORKING_GROUP.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+Upcoming IBC working group calls (Dial in details are provided in the issue for the upcoming meeting)
+
+- November 14th, 2019 17:00 UTC 
+- November 28th, 2019 17:00 UTC
+- December 12th, 2019 17: 00 UTC 
+
+What is the IBC working goup:
+Currently the IBC working group is being actively led and managed by Tendermint (All in Bits), Agoric and the Interchain Foundation. Over time, we anticipate this will shift into breakout working groups and a more decentralized way of managing the interoperability standards developement process. 
+
+How is works:
```
Comment (cwgoes) [inline IBC_WORKING_GROUP.md]:
```suggestion
How it works:
```

File: IBC_WORKING_GROUP.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+Following each IBC working group meeting, an new issue for the next meeting will be created with a proposed agenda for comments and collaboration from the larger working group. During the two week timeframe between calls, we invite IBC working group members to contribute to agenda  and  join the bi-weekly call to discuss the topics in conjunction with the founding IBC team. Upon concluding a meeting, a recording will be posted and the details and notes will be published in the associated issue.
```
Comment (cwgoes) [inline IBC_WORKING_GROUP.md]:
```suggestion
Following each IBC working group meeting, an new issue for the next meeting will be created with a proposed agenda for comments and collaboration from the larger working group. During the two week timeframe between calls, we invite IBC working group members to contribute to the next meeting's agenda and  join the bi-weekly call to discuss the topics in conjunction with the founding IBC team. Upon concluding a meeting, a recording will be posted and the details and notes will be published in the associated issue. 
```

File: IBC_WORKING_GROUP.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+- November 28th, 2019 17:00 UTC
+- December 12th, 2019 17: 00 UTC 
+
+What is the IBC working goup:
+Currently the IBC working group is being actively led and managed by Tendermint (All in Bits), Agoric and the Interchain Foundation. Over time, we anticipate this will shift into breakout working groups and a more decentralized way of managing the interoperability standards developement process. 
+
+How is works:
+Following each IBC working group meeting, an new issue for the next meeting will be created with a proposed agenda for comments and collaboration from the larger working group. During the two week timeframe between calls, we invite IBC working group members to contribute to agenda  and  join the bi-weekly call to discuss the topics in conjunction with the founding IBC team. Upon concluding a meeting, a recording will be posted and the details and notes will be published in the associated issue. 
+
+The founding IBC team
```
Comment (cwgoes) [inline IBC_WORKING_GROUP.md]:
```suggestion
The founding IBC team:
```

File: IBC_WORKING_GROUP.md [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+
+- Mark S. Miller, Chief Scientist at Agoric
+- Chris Goes, Research and Engineering at Tendermint/Cosmos contributor
```
Comment (cwgoes) [inline IBC_WORKING_GROUP.md]:
```suggestion
- Christopher Goes, Research and Engineering at Tendermint/Cosmos contributor
```

File: IBC_WORKING_GROUP.md [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+- Joon Yun, Research and Engineering at Tendermint/Cosmos contributor
+- Dean Tribble, CEO of Agoric
+- Ethan Buchman, Technical Director at the Interchain Foundation
+- Zaki Manian, Head of Research at Tendermint/Cosmos contributor
+- Chris Hibbert, Software Developer at Agoric
+- Brian Warner, Software Developer at Agoric
+- Jae Kwon, Founder of Tendermint and Cosmos Network
+- Anca Zamfir, Distributed systems engineer at the Interchain Foundation
+
+
```
Comment (cwgoes) [inline IBC_WORKING_GROUP.md]: Delete these empty lines?

**Pull request merged**